### ISSUE TITLE
ttag-cli version 1.5.0 and up not working

### ISSUE THREAD

**Issue by matpen** [ttag-org/ttag#194]
I am just approaching `ttag` for the first time (thank you for the nice library, by the way) and setting up some experiments. I just noticed some strange behavior with `ttag-cli`: it seems like all versions from `1.5.0` to `1.8.0` fail to extract translations, while `1.3.0-1` and `1.4.0` work correctly.

For this experiment I am using the [quickstart example](https://ttag.js.org/docs/quickstart.html), which I clone directly from github, but I tried with other simple JavaScript files, with same results:
```bash
# Clone the repo to test the example
cd /tmp
git clone https://github.com/ttag-org/ttag.git
cd ttag/examples/quickstart

# Test version 1.4.0: this works correctly
npm i ttag-cli@1.4.0 \
    && ./node_modules/.bin/ttag extract counter.js \
    && cat translations.pot

# command output:
# [ ... ]
# + ttag-cli@1.4.0
# added 218 packages from 119 contributors in 17.49s
# [ ... ]
# ✔ [ttag] translations extracted to translations.pot
# ... output of cat follows (file looks correct)
# msgid ""
# msgstr ""
# "Content-Type: text/plain; charset=utf-8\n"
# [ ... ]

# Test version 1.5.0: this does not work, the file is empty
npm i ttag-cli@1.5.0 \
    && ./node_modules/.bin/ttag extract counter.js \
    && cat translations.pot

# command output:
# [ ... ]
# + ttag-cli@1.5.0
# added 315 packages from 135 contributors and audited 2604 packages in 10.746s
# [ ... ]
# ✔ [ttag] translations extracted to translations.pot
# ... no more output from cat, file is empty
```

**Comment by AlexMost** [comment]
Hi @matpen! Thanks for the report, need to check that.

**Comment by matpen** [comment]
My pleasure! Thanks you for the attention!

**Comment by AlexMost** [comment]
I guess the trouble is in `require`. If you will change `require` to `import` - it works. Will try to figure out what is wrong with `requrie` discoverage.

**Comment by matpen** [comment]
This is an interesting find. Indeed, if we change the extract command to
```
./node_modules/.bin/ttag extract --discover=t --discover=ngettext --discover=msgid counter.js
```
it works even with the latest version of `ttag-cli`.

So the problem must be somewhere in the interaction of the babel plugin with `require`.

**Comment by AlexMost** [comment]
fixed in `ttag-cli@1.8.1`. Have checked that locally, seems like it works. Thanks for the report and, please, let me know if you will encounter some more issues!

**Comment by matpen** [comment]
I just tested `1.8.2` and it seems to work correctly! Thank you for working on that!
